apps/sync/test/unit/sync-engine/kinto-mock.js

```diff
@@ -213,8 +230,12 @@ empting to fetch resource.`));
     };
   };
 
+  Kinto.syncCount = {};
```

Comment:
This is a static property, so counts add up globally - doesn't matter how often you instantiate `new Kinto()` or how often you call `myKintoInstance.collection()`.